modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -10202,7 +10202,7 @@ export const console: ConsoleNS = {
                 },
                 placeholders: {
                     emptyList: {
-                        action: "Add Organization",
+                        action: "New Organization",
```

Comment:
Don't we have these translations in other languages?

Reply:
fixed